package exceptionHandling;

class CustomException extends Exception {
	public CustomException(String mess) {
		super(mess);
	}
}

public class OwnException {

	public static void StudentAge(int age) throws CustomException {
		if (age < 0)
			throw new CustomException("AgeLessThanZeroException");
		else
			System.out.println("Valid Age");
	}

	public static void main(String[] args) throws CustomException {
		Scanner input=new Scanner(System.in)
		try {
			StudentAge(2);
		} catch (CustomException ex) {
			System.err.println("The exception is " + ex);
		}
	}

}
